about/supported-hardware.md

```diff
@@ -21,8 +21,8 @@ are welcome.
 | Series  | GPU            | Driver      | WPE Backend | Cog Shells |
 |---------|----------------|-------------|-------------|------------|
 | i&period;MX 51 | Imageon Z460   |             |     | |
-| i&period;MX 6  | Vivante GC2000 | Proprietary | fdo | fdo |
-| i&period;MX 8M | Vivante GC7000 | Proprietary | fdo | fdo |
+| i&period;MX 6  | Vivante GC2000 | Vivante (Proprietary) & etnaviv (reverse-engineered) | fdo | fdo |
+| i&period;MX 8M | Vivante GC7000 | Vivante (Proprietary) & etnaviv (reverse-engineered) | fdo | fdo |
```

Comment:
IMO it would be better to have a separate row in the table for the Etnaviv driver, as done for example for the Raspberry Pi, because each driver may also have different WPE Backends that work for it.